test/fixtures/policy/dependencies/dependencies-missing-export-policy.json

```diff
@@ -0,0 +1,11 @@
+{
+  "resources": {
+    "../bad-main.mjs": {
+      "integrity": true,
+      "dependencies": true
+    },
+    "../dep.js": {
+      "integrity": true
+    }
+  }
+}
```

Comment:
nit: Worth adding the trailing new line?

Reply:
```suggestion
}

```

Reply:
done